lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -312,6 +312,7 @@
     <datatype extension="gii" auto_compressed_types="gz" type="galaxy.datatypes.images:Gifti" mimetype="application/octet-stream" display_in_upload="true" />
     <datatype extension="tck" type="galaxy.datatypes.images:Tck" mimetype="application/octet-stream" display_in_upload="true" />
     <datatype extension="trk" type="galaxy.datatypes.images:Trk" mimetype="application/octet-stream" display_in_upload="true" />
+    <datatype extension="mrc" type="galaxy.datatypes.images:Mrc2014" mimetype="application/octet-stream" display_in_upload="true" />
```

Comment:
The extension and the test dataset filename need to match.

Reply:
Ah, sorry I missed that.  I started out with the longer extension and then found that the shorter extension is the one that is used.